snippets/hooks.json

```diff
@@ -3548,56 +3518,13 @@
     ],
     "scope": "php"
   },
-  "hook_ckeditor_plugin_info_alter": {
-    "prefix": "hook_ckeditor_plugin_info_alter",
-    "body": [
-      "/**",
-      " * Implements hook_ckeditor_plugin_info_alter().",
-      " */",
-      "function ${1:${TM_FILENAME_BASE:hook}}_ckeditor_plugin_info_alter(array &$plugins) {",
-      "  $0",
-      "}"
-    ],
```

Comment:
hook_ckeditor_plugin_info_alter should still exist, as [it is present in 9.5](https://git.drupalcode.org/project/drupal/-/blob/9.5.x/core/modules/ckeditor/ckeditor.api.php#L26-28)